docs/using/installing/index.rst

```diff
@@ -379,6 +396,8 @@ Copy the generated files to ``/etc/flocker/`` on the target node and name them `
 Perform the same ``chmod 600`` commands on ``node.key`` as you did for the control service in the instructions above.
 The ``/etc/flocker/`` directory should be set to ``chmod 700``.
 
+You should now have ``cluster.crt``, ``node.crt``, and ``node.key`` on each of your agent nodes, and ``control-service.crt`` and ``control-service.key`` on your control node.
```

Comment:
Add `cluster.crt` for control node as well, as it is needed in the case that the control node is not an agent node (and is merely redundant if the control node is an agent node).